workflow/cases/coupled_free_forecast_s2s.yaml

```diff
@@ -0,0 +1,53 @@
+case:
```

Comment:
I am assuming the _s2s means running with waves?  There are no updates in the README that correspond to running a different case file.

Reply:
@JessicaMeixner-NOAA Yes this is the dedicated case file for wave-included run. I haven't managed to update the README yet. I plan to update it after we could generally agree on these changes.